controllers/users/_preview_scoreboard.htm

```diff
@@ -18,7 +18,7 @@ <h4><?= e(trans('rainlab.user::lang.user.joined')) ?></h4>
             <?= $formModel->created_at->toFormattedDateString() ?>
         </p>
         <p class="description">
-            Status:
+            <?= e(trans('rainlab.user::lang.user.status_text', ['status' => trans('rainlab.user::lang.user.status_guest')])); ?>:
```

Comment:
This would need to be inside of each conditional block and nothing would be needed outside of the if block